source/com.microsoft.tfs.core/src/com/microsoft/tfs/core/telemetry/TfsTelemetryHelper.java

```diff
@@ -44,8 +50,21 @@
             TfsTelemetryInstrumentationInfo.isDeveloperMode());
     }
 
+    public synchronized static void checkDisableTelemetryProperty() {
+        // Unfortunately, we can't push the disabled logic onto the
+        // TelemetryClient itself because the initialization code for
+        // that object simply takes too much time. So, we have to stop
+        // the sending of telemetry here in the helper class.
+
+        // If the system property exists or the env var is true, then we disable
+        // telemetry.
+        sendingTelemetryDisabled = System.getProperty(DISABLE_TELEMETRY_PROPERTY_NAME) != null
```

Comment:
Would a "-Dcom.microsoft.tfs.core.telemetry.disabletelemetry=false" still disable it anyway?  I guess we are using this as a flag instead of a property?